plugins/woocommerce/includes/class-wc-structured-data.php

```diff
@@ -234,7 +234,6 @@ public function generate_product_data( $product = null ) {
 						'priceSpecification' => array(
 							array(
 								'@type'                 => 'UnitPriceSpecification',
-								'priceType'             => 'https://schema.org/ListPrice',
```

Comment:
While this wasn't causing an error, according to Google:

> Don't mark the current, offer price with a priceType property.

https://developers.google.com/search/docs/appearance/structured-data/merchant-listing#sale-pricing-example